### ISSUE TITLE
Cover icon not display if first image is TIFF

### ISSUE THREAD

**Issue by mikemcduffie** [pkrll/ComicZipper-2#36]
2.5.1 (043C): 

If the first image is type TIFF, the next non-TIFF image is displayed. 

If no non-TIFF Images exist, no cover image is displayed.

I'm not sure about all comic viewers, but Simple Comic supports TIFF image files. 

*Note: GIF (including animated although the animation is slow — cool though) and PNG files work.*

**Comment by pkrll** [comment]
Yes, the problem was that I had a list of valid file extensions, where TIFF was not included. This is fixed now. 

But Is there any other type of images that should be included, besides of course jpg, gif, png and tiff?

**Comment by mikemcduffie** [comment]
Again I haven't looked at the code, but I tested for the alternate forms (jpeg, tif/tiff) and the ALL CAPS versions. The ones I tested worked so I assume your handling character case, etc.

The only other common image format I can think of is BMP. (Not that anything except jpg/jpeg is all that common.)

WEBP is a new one, but  it's tied to the CBT file extension to my knowledge. Nice compression though. There's non-Google high space efficiency image format I saw recently but I doubt either of these will be common for some time.
